utils/freshmeat/lib_projxml.h.php

```diff
@@ -74,10 +74,10 @@ function get_entry($in_handle, $marker){
   while( false != ($line = fgets($in_handle, 1024))){
     // </project> is the end tag, save it and return, all done.
     if (preg_match('|</project>|', $line)){
-      array_push(&$project, $line);
+      array_push($project, $line);
       return($project);
     }
-    array_push(&$project, $line);
+    array_push($project, $line);
```

Comment:
Can be optimized for single insertion:
```suggestion
    $project[] = $line;
```

Reply:
Oops, looks like you missed this suggestion :-)